src/color-system.mdx

```diff
@@ -0,0 +1,130 @@
+
+## Usage
+
+**Note:** *Work in progress!* We are currently in the process of transitioning Lime Elements to eventually fully support dark mode. During this process, we recommend that any clients using Lime Elements load `@limetech/lime-elements/dist/lime-elements/style/color-palette-extended-light-mode-only.css`, where dark mode support is completely disabled, to avoid inconsistent behavior. To get dark mode support, to the extent available at the time, load `@limetech/lime-elements/dist/lime-elements/style/color-palette-extended.css` instead.
+
+### Using CSS variables
+We have not used HEX values to create the CSS variables for each color. They are instead written in RGB digits, without parentheses. For example `--color-blue-default: 33, 150, 243;` (which would be `#2196f3` in HEX). The reason for such a choice is empowering you to write the colors with alpha channel if needed.
+
+Thus, to get a solid blue color, you would write `color: rgb(var(--color-blue-default));`.  
+To get a blue color with 10% opacity, write `color: rgba(var(--color-blue-default), 0.1);` instead.
+
+### Customizing our components and your apps with colors
+When using web-components from Lime Elements, you will notice that there are many components which allow you to add colors. However, since Lime Elements is a design system intended to unify the look and feel of our products, we naturally try to reduce your styling choices. So this color customization will enable you to either add an accent color (e.g. your product color) to the component, or add colors that contribute the the functionality of the component (e.g. indicating a status such as error, danger, etc).
+
+But when you do your own add-ons, integrations, and apps, keep in mind that the UI should be neutrally colored (using the contrast scales). Applying intensive colors to big areas of the UI (such as headers and background colors) is not recommended.
+
+Colors should be used to communicate meaning, and attract attention to things that are functionally important, not to scream your brand throughout the entire product. Different sections of the app should be distinct, but also blend together well. Extensive color usage will be placing the emphasis on an app’s UI structure, instead of its content.
+
+Use only one accent color (product color) and use it for all Lime Elements components in your product. Define this accent colour by setting the CSS variable `--lime-primary-color`. We don't recommend using the Lime brand color CSS variables directly in this case, but instead suggest using their equivalent swatch from the color palette. This will ensure that your UI will automatically benefit from dark mode adjustments, while users do not notice the difference.
```

Comment:
```suggestion
Use only one accent color (product color) and use it for all Lime Elements components in your product. Define this accent color by setting the CSS variable `--lime-primary-color`. We don't recommend using the Lime brand color CSS variables directly in this case, but instead suggest using their equivalent swatch from the color palette. This will ensure that your UI will automatically benefit from dark mode adjustments, while users do not notice the difference.
```

Reply:
![image](https://user-images.githubusercontent.com/154725/91304766-51242000-e7aa-11ea-8218-e02367f0196b.png)

Yeah… I spell it "colour" (aka *correctly*) myself, but since we already use `color` everywhere in this document. Also, programming is done in American English (always, no exceptions), and while I'm happy to spell things differently in code and elsewhere, I don't expect anyone else to cope with such an existence 😂